neurons/validator.py

```diff
@@ -17,17 +17,12 @@
 # OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 # DEALINGS IN THE SOFTWARE.
 
-import asyncio
 import time
-import os
-import requests
 import random
 # Bittensor
 import bittensor as bt
+import numpy as np
 
-import base64
-import io
-import torch
```

Comment:
Great job getting rid of unused libraries

Reply:
:ok_hand: